tmt/steps/provision/mrack.py

```diff
@@ -1363,8 +1372,8 @@ def reboot(
         :returns: ``True`` if the reboot succeeded, ``False`` otherwise.
         """
 
-        if not command and hard:
-            self.debug("Reboot using the reboot command 'bkr system-power --action reboot'.")
+        if hard:
+            self.debug("Hard reboot using the reboot command 'bkr system-power --action reboot'.")
```

Comment:
According to https://beaker-project.org/docs/man/bkr-system-power.html
```suggestion
            self.debug("Hard reboot using the reboot command 'bkr system-power --action reboot --force'.")
```

Reply:
> Normally this command will refuse to run if another user is using the system. Pass this option to perform the power action anyway.

I don't believe `--force` would make sense: if we do not own the system, we should probably stop using it rather than forcing a reboot.

Reply:
> if we do not own the system

This is  mrack (not connect/local) plugin, we would own the system if would us who provision it.
and I thought hard reboot mean we want to reboot the system anyway.

However, I don't have a strong opinion on this, and I vote for what you professional would choose^^

Reply:
I think the difference is the key: `--force` does not mean just "force (hard) reboot of a machine", but "force (hard) reboot of a machine *used by someone else but you*", and I don't believe we want that amendment clause.